packages/shared/src/hooks/usePostModalNavigation.ts

```diff
@@ -7,7 +7,15 @@ import { FeedItem, PostItem, UpdateFeedPost } from './useFeed';
 import { useKeyboardNavigation } from './useKeyboardNavigation';
 import { Origin } from '../lib/analytics';
 
+export enum PostPosition {
+  First = 'first',
+  Only = 'only',
```

Comment:
I think the only should not be necessary.

Reply:
It happens when there is only 1 post

Reply:
Ah, right. I am incorrect. Let me resolve this comment 🚢